bitnami/mastodon/templates/tootctlMediaManagement/cronjob.yaml

```diff
@@ -0,0 +1,120 @@
+{{- if .Values.tootctlMediaManagement.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  annotations:
+  name: {{ template "mastodon.tootctlMediaManagement.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: mastodon
+    app.kubernetes.io/component: tootctlMediaManagement
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+spec:
+  concurrencyPolicy: Allow
```

Comment:
Hi,
  I am not into the inners of mastodon, so I am not sure if this job could be run in parallel. 
  Could you confirm it ? My concerns are mostly regarding the deleting of the same object by different job instances and the error handling of that.

Reply:
Multiple instances of the same job can be run in parallel.  Ideally this CronJob is scheduled daily.  In my testing I’ve seen large cleanups (Gigabytes removed on the first run using External S3) take on the order of minutes.  After the initial run the job usually takes seconds to run when all cacheable objects are configured to remove.  .spec.concurrencyPolicy.Allow is the default value if not specified for a CronJob but it could be made configurable.